sass/style.css

```diff
@@ -45,6 +57,33 @@ a {
   background: white;
 }
 
+.search-filter-container .bg-dark-blue {
+  padding: 2rem !important;
+}
+
+.search-filter-container h4 {
+  font-weight: 700;
+  font-size: 22px;
+}
+
+.search-filter-container p {
+  font-size: 16px;
+}
+
+.search-filter-container .input-group-text {
+  padding-right: .7rem !important;
```

Comment:
Would be better to remove the p-2 class from the element and specify the complete padding here, instead of having an !important battle.